crates/wdk-build/src/cargo_make.rs

```diff
@@ -564,7 +563,7 @@ pub fn setup_path() -> Result<impl IntoIterator<Item = String>, ConfigError> {
         .to_string();
     prepend_to_semicolon_delimited_env_var(PATH_ENV_VAR, host_windows_sdk_version_tool_path);
 
-    Ok([PATH_ENV_VAR].map(string::ToString::to_string))
+    Ok([PATH_ENV_VAR].map(std::string::ToString::to_string))
```

Comment:
@svasista-ms This is inconstent with line 496. We should use `ToString::to_string` at both places.